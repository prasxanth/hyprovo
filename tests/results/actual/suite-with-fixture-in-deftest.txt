2021-11-24 22:51:24.789 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-add-variable): sum should be 2
2021-11-24 22:51:24.792 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-add-variable): sum should be 3
2021-11-24 22:51:24.797 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-sub-variable): difference should be 0
2021-11-24 22:51:24.799 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-sub-variable): difference should be 4
2021-11-24 22:51:24.803 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-mul): mul pass
2021-11-24 22:51:24.805 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-mul): mul fail
2021-11-24 22:51:24.811 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-div): div pass
2021-11-24 22:51:24.813 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-div): div fail
2021-11-24 22:51:24.814 - DEBUG - (suite-with-fixture-in-deftest / deftest-with-fixture): Test Summary → Total: 8    Pass: 4    Fail: 4
