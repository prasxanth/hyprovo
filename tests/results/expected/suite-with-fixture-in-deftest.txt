2021-11-24 22:50:31.649 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-add-variable): sum should be 2
2021-11-24 22:50:31.652 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-add-variable): sum should be 3
2021-11-24 22:50:31.658 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-sub-variable): difference should be 0
2021-11-24 22:50:31.661 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-sub-variable): difference should be 4
2021-11-24 22:50:31.666 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-mul): mul pass
2021-11-24 22:50:31.668 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-mul): mul fail
2021-11-24 22:50:31.673 - INFO - Pass ✓... (suite-with-fixture-in-deftest / deftest-with-fixture / test-div): div pass
2021-11-24 22:50:31.676 - ERROR - Fail ✗... (suite-with-fixture-in-deftest / deftest-with-fixture / test-div): div fail
2021-11-24 22:50:31.676 - DEBUG - (suite-with-fixture-in-deftest / deftest-with-fixture): Test Summary → Total: 8    Pass: 4    Fail: 4
